yb-voyager/src/queryissue/queryissue.go

```diff
@@ -95,6 +95,9 @@ func (p *ParserIssueDetector) GetDMLIssues(query string) ([]issue.IssueInstance,
 	if err != nil {
 		return nil, fmt.Errorf("error parsing query: %w", err)
 	}
+	if queryparser.IsViewObject(parseTree) || queryparser.IsMviewObject(parseTree) {
```

Comment:
I think we need to think about this a bit more. What about all the other types of DDLs? 
What if we have a CREATE TABLE with the default of some column value being an xml function. If that is present in the pg_stat_statements, that will come up... 

The only thing that comes to mind right now is that we have a function `isDDL` that detects all types of DDLs, but ofc that is also not fool-proof.